oidc/__init__.py

```diff
@@ -2,6 +2,5 @@
 Main init file for oidc app
 """
 
-VERSION = (0, 0, 10)
+VERSION = (1, 1, 0)
```

Comment:
```suggestion
VERSION = (1, 0, 0)
```

Reply:
Do we use this anywhere ? I believe it's safe to remove if we don't

Reply:
Can be removed, it was initially used in setuptools